src/components/LogView/FullLogLine.js

```diff
@@ -74,6 +74,8 @@ export default class FullLogLine extends React.PureComponent<Props> {
           lineRefCallback={this.props.lineRefCallback}
           text={this.props.line.text}
           lineNumber={this.props.line.lineNumber}
+          toggleBookmark={this.props.toggleBookmark}
```

Comment:
This line doesn't do anything - this.props.handleDoubleClick is already set to https://github.com/evergreen-ci/lobster/blob/master/src/components/LogView/index.js#L206 which calls toggleBookmark already, so you can delete this